### ISSUE TITLE
Add an admin panel

### ISSUE THREAD

**Issue by mithun2003** [benavlabs/FastAPI-boilerplate#87]
Can add an admin panel to this application that displays all user,  posts etc?

**Comment by igorbenav** [comment]
It's a work in progress. I'm cooking something up for the past month and I think in a couple of weeks it will be ready!

**Comment by ahmadsuhail1** [comment]
Any updates bro?

**Comment by FloareDor** [comment]
> It's a work in progress. I'm cooking something up for the past month and I think in a couple of weeks it will be ready!

would loveee to listen to any news on this.

thanks a lot for your work, man.

**Comment by igorbenav** [comment]
Yeah, I didn't have the time to actually finish the admin panel I was working on, so I think I'll just add [sqlalchemy admin](https://github.com/aminalaee/sqladmin) support on the weekend.

**Comment by igorbenav** [comment]
Took a while, but [CRUDAdmin](https://github.com/benavlabs/crudadmin) is coming!

**Comment by igorbenav** [comment]
@arab0v said he'll do it
